doc/input-output-reference/src/input-for-output.tex

```diff
@@ -346,6 +346,21 @@ \subsubsection{Inputs}\label{inputs-7-021}
 OutputControl:ReportingTolerances,.21,.21;
 \end{lstlisting}
 
+\subsection{OutputControl:ResilienceSummaries}\label{OutputControlResilienceSummaries}
+
+This input specifies methods or algorithms used in certain resilience reporting variables.
+
+\subsubsection{Inputs}\label{inputs-8-020}
```

Comment:
When viewing the changes on github, there is a warning about this line:
`Label `inputs-8-020' multiply defined.`

Reply:
let me check on why this happens

Reply:
I see, my label for the inputs section in my latex documentation collides with a existing one. I just changed the label.